### PR TITLE
Added cool code contribution

### PR DESCRIPTION
**Pull request by sfanetti**
This is my pull request.

### CODE REVIEW

**Review by Thomas-Christian — APPROVED** [review]

File: Readme [hunk 1]
```diff
@@ -1 +1,4 @@
 
+kqjrfe lkjash alkjsdf alkjdk dalkjsdh alkjsd halkjsdhsfsdhjasd f
+
```
Comment (sfanetti) [inline Readme:2]: I think the line above may not compile.

File: Readme [hunk 2]
```diff
@@ -1 +1,4 @@
...
+This is my controbution to meep meep.
```
Comment (sfanetti) [inline Readme:3]: Please be more concise with your comments.

**Review by sfanetti — COMMENTED** [review]
I have some changes I want you to make.

File: Readme (hunk 1 above)
Comment (sfanetti) [reply Readme:2]: Bro - I did it

File: Readme (hunk 2 above)
Comment (sfanetti) [reply Readme:3]: I don't want to listen to you anymore

**Review by Thomas-Christian — APPROVED** [review]

**Comment by Aeverette0120** [comment]
Accidentally merged branches. Sorry

**Pull request merged**